eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/TranslationCollectorPass.php

```diff
@@ -21,19 +21,20 @@ class TranslationCollectorPass implements CompilerPassInterface
     public const LOCALES_MAP = [
         'de_DE' => 'de',
         'el_GR' => 'el',
+        'en_US' => 'en',
         'es_ES' => 'es',
         'fi_FI' => 'fi',
         'fr_FR' => 'fr',
         'hi_IN' => 'hi',
+        'hr_HR' => 'hr',
         'hu_HU' => 'hu',
+        'it_IT' => 'it',
         'ja_JP' => 'ja',
-        'nb_NO' => 'no',
+        'nb_NO' => 'nb',
```

Comment:
Besides bugfix mentioned in PR description, I've changed the mapping of `nb_NO` to `nb` (and added `no_NO`).